### PR TITLE
First and last buttons

### PR DESCRIPTION
**Pull request by armantark**
Code for first and last buttons. It is dependent on code in the JS app.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -141,6 +141,13 @@
     t.index ["nct_id"], name: "index_wiki_pages_on_nct_id", unique: true
   end
 
+  create_table "word_frequencies", force: :cascade do |t|
```
Comment (ericharding) [inline db/schema.rb]: Does this need to be here? Is that the table for type ahead?

Comment (armantark) [reply db/schema.rb]: I didn't put that there... Seems like it was from a merge that Nick did

Comment (ericharding) [reply db/schema.rb]: I'm pretty sure it doesn't belong here.  Once you take it out I'll merge this PR.

Comment (armantark) [reply db/schema.rb]: I removed it.

**Comment by ericharding** [comment]
Thanks.

**Pull request merged**